proto/initia/tx/v1/query.proto

```
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package initia.tx.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
```

buf

import "cosmos/base/v1beta1/coin.proto": file does not exist

(COMPILE)